hc/api/models.py

```diff
@@ -49,6 +49,8 @@ class Meta:
     created = models.DateTimeField(auto_now_add=True)
     timeout = models.DurationField(default=DEFAULT_TIMEOUT)
     grace = models.DurationField(default=DEFAULT_GRACE)
+    priority = models.IntegerField(default=0)
+    escalation_emails = models.CharField(blank=True, null=True, max_length=500)
```

Comment:
A `max_length` of 500 is an overkill!
I don't think there's any email that long. consider 100 or 140 chars instead.

Reply:
Had initially thought of using a list of emails as escalation emails thus the max_length of 500 :) Changed to 140 now